docs/source/seth_developers_guide/cli_reference.rst

```diff
@@ -0,0 +1,120 @@
+..
+   Copyright 2017 Intel Corporation
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+
+*************
+CLI Reference
+*************
+
+.. _seth-cli-reference-label:
+
+Seth Client Usage
+=================
+
+seth
+----
+
+.. literalinclude:: /cli/output/seth_usage.out
+   :language: console
+   :linenos:
+
+seth init
+---------
+.. literalinclude:: /cli/output/seth_init_usage.out
+   :language: console
+   :linenos:
+
+seth show
+---------
+.. literalinclude:: /cli/output/seth_show_usage.out
+   :language: console
+   :linenos:
+
+seth account
+------------
+.. literalinclude:: /cli/output/seth_account_usage.out
+   :language: console
+   :linenos:
+
+seth account import
+-------------------
+.. literalinclude:: /cli/output/seth_account_import_usage.out
+   :language: console
+   :linenos:
+
+seth account list
+-----------------
+.. literalinclude:: /cli/output/seth_account_list_usage.out
+   :language: console
+   :linenos:
+
+seth account create
+-------------------
+.. literalinclude:: /cli/output/seth_account_create_usage.out
+   :language: console
+   :linenos:
```

Comment:
You probably didn't touch "seth account create" in this PR, but I wanted to make a note that the defaults for --nonce and --wait might be wrong (you mentioned this in our conversation last wee).

Reply:
Updated